analysisOnData/data/systematics.py

```diff
@@ -1,6 +1,8 @@
 systematics = {
     #"puWeight"  : (["puWeightUp", "puWeightDown"],"puWeightVars"),
+    "PrefireWeight"  : (["PrefireWeight_Up", "PrefireWeight_Down"],"PrefireWeightVars"),
```

Comment:
can you change PrefireWeight_Up -> PrefireWeightUp and down also?

Reply:
yes can be done